src/createFirebaseInstance.js

```diff
@@ -407,7 +429,8 @@ export const createFirebaseInstance = (firebase, configs, dispatch) => {
     watchEvent,
     unWatchEvent,
     reloadAuth,
-    linkWithCredential
+    linkWithCredential,
+    connect
```

Comment:
What about a different name for this export since `react-redux` uses `connect` and people commonly integrate with that.

Reply:
Sure, happy to change it. How about firebaseConnectHelper or connectToEvents (to match watchEvents)?